helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -658,6 +659,17 @@ private RollingUpdateOp rollingUpdateAwaitRunning(final ZooKeeperClient client,
         metadata.put("jobState", taskStatus.getState());
         metadata.put("previousJobStates", previousJobStates);
         metadata.put("throttleState", taskStatus.getThrottled());
+
+        if (taskStatus.getThrottled().equals(ThrottleState.IMAGE_MISSING)) {
```

Comment:
fairly minor, but just in case you weren't aware for enums you can also use `==` for comparisons